### ISSUE TITLE
Backward compatibility for empty sketches generated from older versions

### ISSUE THREAD

**Issue by a2l007** [apache/incubator-datasketches-java#269]
The latest version of sketches doesn't seem to be backward compatible for empty sketches generated using older versions. For example in an union operation, using an empty sketch such as `AQMDAAAazJM=` generated using the older versions would throw the error:
```
java.lang.AssertionError: reqOffset: 8, reqLength: 8, (reqOff + reqLen): 16, allocSize: 8

    at com.yahoo.memory.UnsafeUtil.assertBounds(UnsafeUtil.java:167)
    at com.yahoo.memory.BaseState.assertValidAndBoundsForRead(BaseState.java:330)
    at com.yahoo.memory.BaseWritableMemoryImpl.getNativeOrderedLong(BaseWritableMemoryImpl.java:284)
    at com.yahoo.memory.WritableMemoryImpl.getLong(WritableMemoryImpl.java:133)
    at com.yahoo.sketches.theta.UnionImpl.update(UnionImpl.java:278)
```

@AlexanderSaydakov @leerho This is the same problem that I had discussed with you and I'm logging this issue for tracking purposes.

**Comment by leerho** [comment]
@a2l007  This should be fixed as of apache version 1.1.0 and later.  Could you please confirm?

Thanks

**Comment by leerho** [comment]
I assume this has been fixed.
